pkgs/configs/src/lib.zig

```diff
@@ -51,52 +51,59 @@ const ChainConfigError = error{
 };
 
 pub fn genesisConfigFromYAML(config: Yaml, override_genesis_time: ?u64) !types.GenesisSpec {
-    const genesis_time: u64 = if (override_genesis_time) |gen_time| gen_time else @intCast(config.docs.items[0].map.get("GENESIS_TIME").?.int);
-    const genesis_spec: types.GenesisSpec = .{
-        .genesis_time = genesis_time,
-        .num_validators = @intCast(config.docs.items[0].map.get("VALIDATOR_COUNT").?.int),
-    };
-    return genesis_spec;
+    _ = config;
+    _ = override_genesis_time;
+    // TODO: Implement YAML parsing for validator pubkeys
+    // This function needs to:
+    // 1. Parse VALIDATOR_COUNT or VALIDATOR_PUBKEYS from config.yaml
+    // 2. If VALIDATOR_COUNT: create keymanager and extract pubkeys
+    // 3. If VALIDATOR_PUBKEYS: read them directly from YAML
+    // 4. Return GenesisSpec with genesis_time and validator_pubkeys populated
+    // Until this is implemented, use beam command for testing or provide genesis_spec programmatically
+    return error.NotImplemented;
 }
 
-test "load genesis config from yaml" {
-    const yaml_content =
-        \\# Genesis Settings
-        \\GENESIS_TIME: 1704085200
-        \\
-        \\# Validator Settings  
-        \\VALIDATOR_COUNT: 9
-    ;
-
-    var yaml: Yaml = .{ .source = yaml_content };
-    defer yaml.deinit(std.testing.allocator);
-    try yaml.load(std.testing.allocator);
-
-    const genesis_config = try genesisConfigFromYAML(yaml, null);
-
-    try std.testing.expect(genesis_config.genesis_time == 1704085200);
-    try std.testing.expect(genesis_config.num_validators == 9);
-
-    const genesis_config_override = try genesisConfigFromYAML(yaml, 1234);
-    try std.testing.expect(genesis_config_override.genesis_time == 1234);
-    try std.testing.expect(genesis_config_override.num_validators == 9);
-}
-
-test "custom dev chain" {
-    const dev_spec =
```

Comment:
also restore such parsing in a followup PR here or in the mock test sequence in the state transition